tests/test_dataprep_redis_langchain_ray.sh

```diff
@@ -63,10 +70,18 @@ print(resp.text)
 assert "name" in resp.text, "Response does not meet expectation."
 print("Request successful!")
 EOF
-)"
-    echo "Validation successful"
+)" || EXIT_CODE=$?
+    rm -rf dataprep_file.txt
+    if [ $EXIT_CODE -ne 0 ]; then
+        echo "[ dataprep ] Validation failed. Entire log as below doc "
+        docker container logs test-comps-dataprep-redis-ray-server | tee -a ${LOG_PATH}/dataprep.log
```

Comment:
hi, you have to define the `${LOG_PATH}` before using it. Otherwise the CI workflows can not catch the test log. Please refer to `test_dataprep_redis_langchain.sh`.
![image](https://github.com/opea-project/GenAIComps/assets/106566639/c8a0865b-8a00-40ea-8fe3-dddf927619a3)

Reply:
Done